Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+
+- **Hide content editor** - Use this feature when you want to hide the ability to edit the content in a Block Editor. If you made a custom view that enables you to edit the content part of a Block, you might want to use this feature.
+
+## Editing Blocks
+
+When viewing a **Block Grid** editor in the Content section for the first time, you will be presented with the option to Add content.
+
+![Block List - Add Content](images/BlockListEditor_AddContent.png)
+> TODO: screenshot
+
+Clicking the Add content button brings up the **Block Catalogue**.
+
+![Block List - Setup](images/BlockListEditor_BlockPicker_simplesetup.jpg)
+> TODO: screenshot
+
+The Block Catalogue looks different depending on the amount of available Blocks and their catalogue appearance.
+
+![Block List - example setup from Umbraco.com](images/BlockListEditor_BlockPicker.jpg)
+> TODO: screenshot
+
+Click the Block Type you wish to create and a new Block will appear in the layout.
+
+More Blocks can be added to the layout by clicking the Add content button. Alternatively, use the Add content button that appears on hover to add new blocks between, besides or above existing Blocks.
+
+![Block List - Add Content](images/BlockListEditor_AddContentInline.jpg)
+> TODO: screenshot
+
+Blocks can be rearranged using the click and drag feature. Move them up or down to place them in the desired order.
+
+To delete a Block click the trashbin icon whichs appears on hover.
+
+## Rendering Block Grid Content
+
+Rendering the stored value of your **Block Grid** property can be done in two ways: 
+
+1.  [Default rendering](#1-default-rendering)
+2.  [Build your own rendering](#2-build-your-own-rendering)
+
+### 1. Default rendering
+
+You can choose to use the built-in rendering mechanism for rendering blocks using a Partial View for each block.
+
+The default rendering method is named `GetBlockGridHtmlAsync()` and comes with a few options.
+
+Example use-case:
+
+```csharp
+@await Html.GetBlockGridHtmlAsync(Model, myGrid")
+```
+
+In the sample above `"myGrid"` is the alias of the Block Grid editor.
+
+If you are using ModelsBuilder, the example will look like this:
+
+Example:
+
+```csharp
+@await Html.GetBlockGridHtmlAsync(Model.MyGrid)
+```
+
+To use the `GetBlockGridHtmlAsync()` method you will need to create a Partial View for each Block Type. The Partial View must be named using the alias of the Element Type that is being used as Content Model for the Block Type.
+
+These Partial View files need to go into the `Views/Partials/BlockGrid/Components/` folder.
+
+Example: `Views/Partials/BlockGrid/Components/MyElementTypeAliasOfContent.cshtml`.
+
+The Partial Views will receive a model of type `Umbraco.Core.Models.Blocks.BlockGridItem`. This model contains `Content` and `Settings` from your block, as well as the configured `RowSpan`, `ColumnSpan`, `ForceLeft`, `ForceRight` and `Areas` of the block.
```

Comment:
Perhaps a screenshot or a codeblock showing what the Model contains @sofietoft & @nielslyngsoe 
For example a screenshot of the debugger with values shown for the properties to help understand what is available to use.

Reply:
I am not entirely sure I understand what you mean, @warrenbuckley ? 🤔 
Which debugger is this, and how do I use it? (perhaps we already have docs on it, but just wanted to check)

Reply:
Sorry @sofietoft I just meant use say Visual Studio debugger with a breakpoint on the razor view and with a watch in the debugger showing some of the values.

Reply:
Ah, alrighty! Perhaps you'd be up for adding something like that? Or perhaps do it, send me the screendump, and I'll add it to the article?
Sorry, I wouldn't really know where to even start using the debugger 🙈